frontend/packages/ui/src/components/Sidebar/Sidebar.tsx

```diff
@@ -396,7 +401,18 @@ const SidebarMenuAction = forwardRef<
 >(({ className, asChild = false, showOnHover = false, ...props }, ref) => {
   const Comp = asChild ? Slot : 'button'
 
-  return <Comp ref={ref} data-sidebar="menu-action" {...props} />
+  return (
+    <Comp
+      ref={ref}
+      data-sidebar="menu-action"
+      className={clsx(
+        styles.sidebarMenuAction,
+        showOnHover && styles.showOnHover,
+        className,
+      )}
+      {...props}
+    />
+  )
```

Comment:
The styling of the SidebarMenuAction component in shadcn has now been implemented.
It is useful to have showOnHover props available 👍🏻 

https://ui.shadcn.com/docs/components/sidebar#sidebarmenuaction